src/lib/_loading.scss

```diff
@@ -38,8 +38,8 @@
     }
 }
 
-// Preview content should not be interactable during loading
-:not(.bp-loaded) > .bp-loading-wrapper + div {
+// Only the loading wrapper should be interactable inside the Preview container
+.bp:not(.bp-loaded) > div:not(.bp-loading-wrapper) {
```

Comment:
Can we avoid 2 nots? Maybe add this to all and then nest for the .bp-loading-wrapper where we turn touch events back on.

Reply:
👍